### ISSUE TITLE
Add UI for setting entry fees for each event

### ISSUE THREAD

**Issue by pingskills** [thewca/worldcubeassociation.org#2444]
All the database structures are in place, we now need a UI to allow for a competition organiser to set the entry fee for individual events.

**Comment by Mollerz** [comment]
Anyone that can start this soon? Maybe @viroulep? :)

Would really appreciate this...

**Comment by viroulep** [comment]
To be honest I'd rather focus on #278 that I started working on before I was totally busy with non cubing stuff, and which I think is high priority as the Board is currently in pain every time they have to announce a competition.

@pingskills do you have any availability to work on the UI?

If not I'll definitely keep it in mind but can't guarantee anything :s

@Mollerz do you have any deadline by which you would like to have this done?
(I know it's not for Euro :p)

**Comment by Mollerz** [comment]
Fair enough, I think that would benefit more people anyway. For the UK we want it as soon as possible, we make the system as simple as possible but it's still awful trying to deal with it all. Would save a few people many hours every week.
